Dockerfile.windows

```diff
@@ -219,15 +216,11 @@ RUN `
   Write-Host INFO: Downloading compiler 3 of 3...; `
   Download-File https://raw.githubusercontent.com/jhowardmsft/docker-tdmgcc/master/binutils.zip C:\binutils.zip; `
   `
-  Write-Host INFO: Installing PS7Zip package...; `
-  Install-Package PS7Zip -Force | Out-Null; `
-  Write-Host INFO: Importing PS7Zip...; `
-  Import-Module PS7Zip -Force; `
-  New-Item C:\git -ItemType Directory | Out-Null ; `
-  cd C:\git; `
   Write-Host INFO: Extracting git...; `
-  Expand-7Zip C:\gitsetup.7z.exe | Out-Null; `
-  cd C:\; `
+  Expand-Archive C:\gitsetup.zip -DestinationPath C:\git-temp; `
```

Comment:
If you want to move to nuget, why not just use `Install-Package` with an appropriate version? Otherwise, this is no different to effectively what's happening now, just you're downloading from an indirect source rather than the master distro location (which arguably is the source of truth for the package)?